nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -319,429 +320,447 @@ def _threshold_and_rescale(data, threshold, vmin, vmax):
-            )
-        if axes is not None and not isinstance(axes, plt.Axes):
+def _get_view_plot_surf(hemi, view):
+    """Check ``hemi`` and ``view``, and return `elev` and `azim` for
+    matplotlib engine.
+    """
+    view = sanitize_hemi_view(hemi, view)
+    if isinstance(view, str):
+        if hemi == "both" and view in ["lateral", "medial"]:
             raise ValueError(
-                "axes argument should be None or a 'matplotlib.pyplot.Axes'."
+                "Invalid view definition: when hemi is 'both', "
+                "view cannot be 'lateral' or 'medial'.\n"
+                "Maybe you meant 'left' or 'right'?"
             )
-
-    def _get_view_plot_surf(self, hemi, view):
-        """Check ``hemi`` and ``view``, and return `elev` and `azim` for
-        matplotlib engine.
-        """
-        view = self._sanitize_hemi_view(hemi, view)
-        if isinstance(view, str):
-            if hemi == "both" and view in ["lateral", "medial"]:
-                raise ValueError(
-                    "Invalid view definition: when hemi is 'both', "
-                    "view cannot be 'lateral' or 'medial'.\n"
-                    "Maybe you meant 'left' or 'right'?"
-                )
-            return MATPLOTLIB_VIEWS[hemi][view]
-        return view
-
-    def _plot_surf(
-        self,
-        surf_mesh,
-        surf_map=None,
-        bg_map=None,
-        hemi=DEFAULT_HEMI,
-        view=None,
-        cmap=None,
-        symmetric_cmap=None,
-        colorbar=True,
-        avg_method=None,
-        threshold=None,
-        alpha=None,
-        bg_on_data=False,
-        darkness=0.7,
-        vmin=None,
-        vmax=None,
-        cbar_vmin=None,
-        cbar_vmax=None,
-        cbar_tick_format="auto",
-        title=None,
-        title_font_size=None,
-        output_file=None,
-        axes=None,
-        figure=None,
-    ):
-        parameters_not_implemented_in_matplotlib = {
-            "symmetric_cmap": symmetric_cmap,
-            "title_font_size": title_font_size,
-        }
-
-        self._check_engine_params(parameters_not_implemented_in_matplotlib)
-
-        # adjust values
-        avg_method = "mean" if avg_method is None else avg_method
-        alpha = "auto" if alpha is None else alpha
-        cbar_tick_format = (
-            "%.2g" if cbar_tick_format == "auto" else cbar_tick_format
+        return MATPLOTLIB_VIEWS[hemi][view]
+    return view
+
+
+def _plot_surf(
+    surf_mesh,
+    surf_map=None,
+    bg_map=None,
+    hemi=DEFAULT_HEMI,
+    view=None,
+    cmap=None,
+    symmetric_cmap=None,
+    colorbar=True,
+    avg_method=None,
+    threshold=None,
+    alpha=None,
+    bg_on_data=False,
+    darkness=0.7,
+    vmin=None,
+    vmax=None,
+    cbar_vmin=None,
+    cbar_vmax=None,
+    cbar_tick_format="auto",
+    title=None,
+    title_font_size=None,
+    output_file=None,
+    axes=None,
+    figure=None,
+):
+    parameters_not_implemented_in_matplotlib = {
```

Comment:
For private functions, I still like to have a one-line docstring saying what it does.